ios/Classes/SwiftPolarPlugin.swift

```diff
@@ -580,6 +580,8 @@ class StreamingChannel: NSObject, FlutterStreamHandler {
       stream = api.startTemperatureStreaming(identifier, settings: settings!)
     case .pressure:
       stream = api.startPressureStreaming(identifier, settings: settings!)
+    case .skinTemperature:
+      stream = api.startSkinTemperatureStreaming(identifier, settings: settings!)
     }
```

Comment:
I was going to say add location streaming here, but apparently it doesn't exist on iOS